builder/node/node_5.7.0/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# published as clever/drone-node:5.7.0
+FROM bradrydzewski/base
+
+WORKDIR /home/ubuntu
+USER ubuntu
+ADD nodejs.sh /etc/drone.d/
+
+RUN /bin/bash -c ". /home/ubuntu/nvm/nvm.sh && nvm install v5.7.0 && npm install npm@2.7.0 -g"
```

Comment:
Why npm 2.7.0? I think 3 offers advantages so while we're upgrading to node 5 it seems like a good time to get this out of way too, since it's the default version included with node 5.